vespalib/src/vespa/vespalib/util/cpu_usage.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "cpu_usage.h"
+#include "require.h"
+
+namespace vespalib {
+
+namespace cpu_usage {
```

Comment:
I prefer vespalib::cpu_usage { }. Less level to track

Reply:
When I add code to the vespalib namespace there will be more close/re-open if we combine them, not sure if that is better...